src/app/pages/traslados/form-traslado/form-traslado.component.ts

```diff
@@ -256,11 +255,11 @@ export class FormTrasladoComponent implements OnInit {
   private validarTercero(): ValidatorFn {
     return (control: AbstractControl): ValidationErrors | null => {
       const valor = control.value;
-      const checkStringLength = typeof (valor) === 'string' && valor.length < 4 && valor !== '' ? true : false;
-      const checkInvalidString = typeof (valor) === 'string' && valor !== '' ? true : false;
-      const checkInvalidTercero = typeof (valor) === 'object' && !valor.Tercero ? true : false;
+      const checkStringLength = typeof (valor) === 'string' && valor.length < 4 && valor !== '';
+      const checkInvalidString = typeof (valor) === 'string' && valor !== '';
+      const checkInvalidTercero = typeof (valor) === 'object' && !valor.Tercero;
       return checkStringLength ? { errorLongitudMinima: true } :
-        checkInvalidString || checkInvalidTercero ? { terceroNoValido: true } : null;
+        (checkInvalidString || checkInvalidTercero) ? { terceroNoValido: true } : null;
```

Comment:
Confirmar/Revisar si...
```suggestion
        ((checkInvalidString || checkInvalidTercero) ? { terceroNoValido: true } : null);
```